### PR TITLE
Add implementation of unary ops

### PR DESCRIPTION
**Pull request by BruceDai**

### CODE REVIEW

File: src/sigmoid.js [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
  */
 export function sigmoid(input) {
-  const output = new Tensor(input.shape);
-  for (let i = 0; i < input.size; ++i) {
-    const x = input.getValueByIndex(i);
-    const y = 1 / (Math.exp(-x) + 1);
-    output.setValueByIndex(i, y);
-  }
-  return output;
+  return unary(input, (x) => 1 / (Math.exp(-x) + 1));
```
Comment (BruceDai) [inline src/sigmoid.js:12]: Here I tried to invoke `exp` of `unary.js` likes following
```js
import {add, div} from './binary.js';
import {Tensor} from './tensor.js';
import {exp, neg, unary} from './unary.js';

export function sigmoid(input) {
  const TENSOR1 = new Tensor([1], [1]);
  return div(TENSOR1, add(exp(neg(input)), TENSOR1));
}
```

these codes look not simple, so I still use `Math.exp`, and same to implementation of `tanh`.

@huningxin What's your opinion?

**Review by BruceDai — COMMENTED** [review]
@huningxin PTAL, thanks.

File: src/leaky_relu.js [hunk 2]
```diff
@@ -9,12 +9,6 @@ import {Tensor} from './tensor.js';
...
 export function leakyRelu(input, options = {}) {
-  const alpha = options.alpha ? options.alpha : 0.01;
-  const output = new Tensor(input.shape);
-  for (let i = 0; i < input.size; ++i) {
-    const x = input.getValueByIndex(i);
-    const y = Math.max(0, x) + alpha * Math.min(0, x);
-    output.setValueByIndex(i, y);
-  }
-  return output;
+  const alpha = options.alpha !== undefined ? options.alpha : 0.01;
```
Comment (BruceDai) [inline src/leaky_relu.js:12]: Here I updated the condition code, when `options.alpha` is set as `0`, the `leayRelu` actually acts `relu`
```
max(0, x) + alpha ∗ min(0, x)
```
, while the return value of `options.alpha ? options.alpha : 0.01` is `0.01`.

Comment (huningxin) [reply src/leaky_relu.js:12]: Good catch. We should use `options.alpha !== undefined` for all the cases.

File: src/sigmoid.js (hunk 1 above)
Comment (huningxin) [reply src/sigmoid.js:12]: Using Math.exp looks good.

**Review by huningxin — APPROVED** [review]
LGTM, thanks @BruceDai !

**Pull request merged**